### PR TITLE
Update PopCash.net.xml

### PR DESCRIPTION
**Pull request by koops76**

### CODE REVIEW

File: src/chrome/content/rules/PopCash.net.xml [hunk 1]
```diff
@@ -1,16 +1,9 @@
...
-	<target host="tb.popcash.net"/>
-	<target host="www.popcash.net"/>
-	<!-- static.popcash.net  cert mismatch -->
-
-	<rule from="^http:" to="https:"/>
-
+<ruleset name="PopCash.net">
+	<target host="popcash.net" />
+	<target host="www.popcash.net" />
+	<target host="static.popcash.net />
```
Comment (cschanaj) [inline src/chrome/content/rules/PopCash.net.xml]: missing `"`

Comment (koops76) [inline src/chrome/content/rules/PopCash.net.xml]: lol

Comment (koops76) [inline src/chrome/content/rules/PopCash.net.xml]: @cschanaj Fixed.

**Comment by koops76** [comment]
@J0WI Request review.

**Pull request merged**